.circleci/config.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - run:
           name: Check Tagged Push
           command: |
-            PKG_VERSION=$(grep VERSION Dockerfile | cut -d' ' -f3)
+            PKG_VERSION=$(grep BUILD_VERSION Dockerfile | cut -d' ' -f3)
```

Comment:
This has to be changed to `PKG_VERSION=$(grep BUILD_VERSION testfile | cut -d'=' -f2)`

Reply:
Good catch, although this really isn't a problem for this PR.  We still have to update the build steps in an upcoming PR (where I was planning on getting this part right), but we might as well fix this now since it's changed.

@cederberg you should just be able to remove that equal sign and let the ENV be assigned that way.  Then, it'll work as is here.  `ENV BUILD_VERSION 1.2.0`

Reply:
Yup, this works too

Reply:
Now fixed.